database/rrd.h

```diff
@@ -164,6 +164,8 @@ struct label {
 };
 
 struct label *create_label(char *key, char *value, LABEL_SOURCE label_source);
+struct label *add_label_to_list(struct label *l, char *key, char *value, LABEL_SOURCE label_source);
+void reload_host_labels();
```

Comment:
Are you not missing an `extern` before this function declaration?

Reply:
I'll try it and change it if necessary, I expect he's tested it.

Reply:
The extern keyword is not needed. Yes I tested it :)